nss-day-cohort-45/tabloidfullstack-giant-gibbons
# Gt view all tags

## Changes Made:
1. When logged in as an admin, the Tag Management option appears in the Nav bar
2. Routing to tagManagement provides a list of current tags 
## How to Check:
1. Fetch gt-viewAllTags
2. Log in using an admin user
3. Select 'Tag Management' from the Nav bar
4. Verify that the 4 tags appear in alphabetical order
